app/workers/asset_manager_create_whitehall_asset_worker.rb

```diff
@@ -1,7 +1,9 @@
 class AssetManagerCreateWhitehallAssetWorker < WorkerBase
   def perform(file_path, legacy_url_path)
     file = File.open(file_path)
-    Services.asset_manager.create_whitehall_asset(file: file, legacy_url_path: legacy_url_path)
+    asset_options = {file: file, legacy_url_path: legacy_url_path}
```

Comment:
Very minor: It's usual to have spaces after the opening brace and before the closing brace of a Hash. I would've expected Rubocop to complain about this.

Reply:
Rubocop did complain! I fixed it but put it in the subsequent commit by mistake. I've now updated this commit.